Dependencies/xr/Source/OpenXR/include/XrContextHelper.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "jsi/jsi.h"
+#include "IXrContext.h"
+
+namespace BabylonReactNative
+{
+    bool TryGetXrContext(facebook::jsi::Runtime& jsiRuntime, const facebook::jsi::Value& xrSession, IXrContext*& xrContext)

```

Comment:
>bool TryGetXrContext [](start = 4, length = 20)

Is this header meant to be the entry point for native modules? If so, perhaps we should use a `#if __has_include("jsi/jsi.h")` guard against the JSI stuff (thinking of a future where we might want an equivalent NAPI version or something).

Reply:
we assume that this header will only be used when jsi is present. It isn't built in any BabylonNative lib currently. this safe guard seems like a cool addition though, so i will look at to add it :)